### PR TITLE
Switch from using a jquery-ui accordion to using a bootstrap accordion for scaffolds.

### PR DESCRIPTION
**Pull request by drgrice1**
Also the javascript and css are moved out of the macro and into separate files (in webwork2) loaded via ADD_JS_FILE and ADD_CSS_FILE.

See the corresponding webwork2 pull request openwebwork/webwork2#1361

Note that there were two motivating factors for this pull request.  The first is that since the update to the latest version of jquery-ui, the arrows on the scaffold headers have problems.  When the arrow points down it turns white and is very hard to see.  The other is part of the goal to remove all jquery-ui usage in webwork.

### CODE REVIEW

**Comment by drgrice1** [comment]
Also note that this can be done with less code and more cleanly with bootstrap version 5 (which is now officially released).  In fact, I have borrowed some of the bootstrap 5 styling to make this look better than bootstrap 2.3.2 does by default.

File: macros/scaffold.pl [hunk 1]
```diff
@@ -321,7 +324,7 @@ sub End {
   Scaffold->Error("Scaffold ended with section was still open") if $self->{current_section};
   my $self = $scaffold;
   push(@{$self->{output}},splice(@$PG_OUTPUT,0));                      # collect any final non-section output
-  $self->open_sections(@{$self->{open}});                              # make the open sections be displayed
+  $self->hide_other_results(@{$self->{open}});                         # hide results of unnopened sections in the results table
```
Comment (Alex-Jordan) [inline macros/scaffold.pl:327]:
```suggestion
  $self->hide_other_results(@{$self->{open}});                         # hide results of unopened sections in the results table
```

Comment (Alex-Jordan) [reply macros/scaffold.pl:327]: Trying Andrew's GitHub tip.

Comment (Alex-Jordan) [reply macros/scaffold.pl:327]: I have a two-part exercise (three answer blanks, then two). If I enter some answers in the first three blanks, even one of them incorrect, then when I click submit, the results table still has five rows. The bottom two rows are grayed out. This is all while logged in as a student. Is this what I should be seeing? The wording here made me think I should only see three rows.

Screen Shot 2021-05-13 at 2 42 27 PM

**Comment by Alex-Jordan** [comment]
This is working fine for me. I just had the observation and question about rows in the results table.

**Comment by drgrice1** [comment]
That is the same behavior as before and that is the wording that was in there before.  It doesn't really hide the rows though, it dims them (sets the opacity to 0.5).

**Comment by drgrice1** [comment]
There is a comment about hiding the rows instead by using display "none".  Se the comments on lines 439 - 443 of the current file (lines 431-435 in this pull request).

**Review by Alex-Jordan — APPROVED** [review]

**Review by pstaabp — APPROVED** [review]
looks good.

**Pull request merged**